lib/Futures/SharedState.h

```diff
@@ -26,9 +26,8 @@
 #include <atomic>
 #include <function2.hpp>
 
-#include "Basics/debugging.h"
-#include "Futures/Try.h"
-#include "Logger/LogMacros.h"
+#include "Assertions/Assert.h"
```

Comment:
With this include, would you not need to link arango_assertions into arango_futures?

Reply:
yup